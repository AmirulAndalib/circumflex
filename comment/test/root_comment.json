{
  "id": 26298314,
  "title": "Ask HN: Is there an IDE for computer vision?",
  "points": 2,
  "user": "awinter-py",
  "time": 1614556974,
  "time_ago": "6 hours ago",
  "type": "link",
  "content": "<p>How are people running their detection pipelines on video and overlaying the results? Is every project writing its own scaffolding for this? Is it mostly &#x27;run command, generate new video&#x27;? Or is there a GUI &#x2F; toolkit that&#x27;s standard?",
  "url": "item?id=26298314",
  "comments": [],
  "comments_count": 0
}