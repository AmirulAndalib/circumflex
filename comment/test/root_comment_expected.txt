Ask HN: Is there an IDE for computer vision? (]8;;item?id=26298314item?id=26298314]8;;)
[2m2 points by awinter-py 6 hours ago â€¢ 0 comments â€¢ ID 26298314[0m
[2mYou are now in 'less' â€¢ Press 'q' to return and 'h' for help[0m

How are people running their detection pipelines on video and overlaying the
results? Is every project writing its own scaffolding for this? Is it mostly
'run command, generate new video'? Or is there a GUI / toolkit that's standard?
--------------------------------------------------------------------------------

